src/models/GOKU.jl

```diff
@@ -101,11 +139,30 @@ function sample(μ::T, logσ²::T, model::LatentDiffEqModel{GOKU}) where T <: Tu
     return ẑ₀, θ̂
 end
 
+@doc raw"""
+    default_layers(model_type, input_dim::Int, diffeq;
+        device = cpu,
+        hidden_dim_resnet = 200, rnn_input_dim = 32,
+        rnn_output_dim = 16, latent_dim = 16,
+        latent_to_diffeq_dim = 200, θ_activation = softplus,
+        output_activation = σ, init = Flux.kaiming_uniform(gain = 1/sqrt(3)))
+Generates default encoder and decoder layers that are to be fed into the LatentDiffEqModel.
+# Arguments
+model\_type: GOKU() or LatentODE()\
+input\_dim: Dimension of input, 28*28 for pendulum data provided\
+diffeq: Differential equation, e.g. Pendulum() for GOKU or NODE(16) for LatentODE
+# Examples
+```julia-repl
+julia> using LatentDiffEq, OrdinaryDiffEq, ModelingToolkit, DiffEqSensitivity, Flux
+julia> include("pendulum.jl")  # Assuming directory is LatentDiffEq/examples/pendulum_friction-less
+julia> encoder_layers, decoder_layers = default_layers(GOKU(), 28*28, Pendulum())
+```
+"""
```

Comment:
Providing the detailed description of the diffeq layer and tutorial, I think that it's better (at least for now) to remove this example, also avoiding the problem of locating the pendulum.jl file (if you have installed the LatentDiffEq package, the path that you mention here will be in the Julia instalation folder, which is not very convenient to cite). I'll figure this out in the future but just for now I prefer to keep it the simplest possible.